client/state/hosting/reducer.js

```diff
@@ -76,6 +78,26 @@ const phpVersion = ( state = null, { type, version } ) => {
 	return state;
 };
 
+const isFetchingWpVersion = ( state = null, { type } ) => {
```

Comment:
Personally I think the default state here should be `false`

Reply:
Updated with 1660102cea